lib/exfmt/ast/to_algebra.ex

```diff
@@ -354,12 +354,40 @@ defmodule Exfmt.Ast.ToAlgebra do
   end
 
   #
-  # Zero arity qualified function calls
+  # Zero arity type calls
+  #   e.g. String.t
+  #
+  def to_algebra({{:., _, [aliases, :t]}, _, []}, ctx) do
+    zero_arity_call_to_algebra(".t", aliases, ctx)
+  end
```

Comment:
I don't think we can safely decide to use parens or not based upon the name of the function as there's nothing stopping someone from defining a normal function called t, and types with names other than t are common.